### PR TITLE
[Merged by Bors] - Add release commands

### PR DESCRIPTION
**Pull request by Techassi**
This PR adds all release related commands.

### CODE REVIEW

File: bins/stackablectl/src/cli/cluster_args.rs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                            self.cluster_cp_nodes,
+                            name,
+                            namespace,
+                        );
+
+                        // Seems like we cannot propagate the error directly using ?
+                        match kind_cluster.create().await {
+                            Ok(_) => Ok(()),
+                            Err(err) => Err(ClusterError::KindClusterError { source: err }),
+                        }
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cli/cluster_args.rs:75]: Yeah I'm not super happy with that either. We could introduce an `CommonClusterArgsError` with one variant being the `KindClusterError`.

File: bins/stackablectl/src/cmds/release.rs [hunk 2]
```diff
@@ -51,53 +76,199 @@ pub struct ReleaseInstallArgs {
...
+            .context(PathOrUrlParseSnafu {})?;
+
+        let cache_home_path = xdg::BaseDirectories::with_prefix(CACHE_HOME_PATH)
+            .context(XdgSnafu {})?
+            .get_cache_home();
+
+        let release_list =
+            ReleaseList::build(&files, (cache_home_path, !common_args.no_cache).into())
+                .await
+                .context(ListSnafu {})?;
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/release.rs]: Improved in 50f0c57

[the remark this answers is not in the document]
Comment (nightkr) [reply bins/stackablectl/src/cmds/release.rs]: I was thinking more along the lines of 2a3fb9c24eb91dbf482f97f191bb8187267315ef

File: bins/stackablectl/src/cli/cluster_args.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cli/cluster_args.rs]: Improved in a33a40e.

File: bins/stackablectl/src/cli/mod.rs [hunk 3]
```diff
@@ -1,6 +1,11 @@
+// Std library
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cli/mod.rs:1]: Removed in de48648.

**Review by nightkr — APPROVED** [review]
bors retry

**Pull request closed without merging**